### PR TITLE
[doc] reorganize files under for English Manual

### LINKED ISSUE #1056: [doc] reorganize  doc structures
1. some the files under nebula/docs/ looks redundant (e.g. nGQL.md, nGQL-tutorial.md, get-started.md, manual-example-graph.md) 

I'll combine them into a new get-started.md under overview/ directory

2. the files in nebula/docs/overview are not well sorted. I'll re-order and re-arrange the files.

3. I'll move  all the files about deployment & build into a  new directory.

### PR DESCRIPTION
**Pull request by whitewum**
1. delete redundant files. Combine related files in the same directory

2. fix markdown lint problems

3. reorganize files structure to the right directory

TODO: 

1. fix error links (in README.md, and others)

2. reorganise the corresponding files in the Chinese directory.

3. combine sst ingest and csv importer (they are in different directory docs/ and tools/)

fix #1056

### CODE REVIEW

File: docs/manual-EN/1.overview/2.quick-start/1.get-started.md [hunk 1]
```diff
@@ -86,92 +51,18 @@ connect to Nebula:
 
 If you have any questions or concerns about the deployment procedures, please do not hesitate to open an issue on [GitHub](https://github.com/vesoft-inc/nebula/issues).
```
Comment (whitewum) [inline docs/manual-EN/1.overview/2.quick-start/1.get-started.md]: move to other parts. 

delete here.

**Comment by Amber1990Zhang** [comment]
Please solve the conflict first. @whitewum

**Comment by whitewum** [comment]
TODO: fix broken links in README

**Comment by dangleptr** [comment]
Jenkins go

**Pull request merged**